### PR TITLE
Sprint 1 Code Cut: Merge Dev -> Master

### PR DESCRIPTION
**Pull request by mjidris**
## Objective
To merge in the new code in `dev` to `master` following the conclusion of sprint 1. 

## Detailed Description
Sprint 1 ran from 10/14-10/23. Pulling in all the new updates into the master branch following the conclusion of sprint 1. 

## Requirements
[Taiga Sprint](https://tree.taiga.io/project/cbrobsto-iron-db/taskboard/first-sprint-949)

### CODE REVIEW

**Comment by CBrobsto** [comment]
I completed a basic test with a fresh install. Other than the warnings accompanying the install, I found no problems.

**Comment by CBrobsto** [comment]
Jest failed some tests, but I believe it failed before the start of this sprint. I verified that I received the same test results in Master, no new issues were shown. 
image

File: public/javascripts/database.js [hunk 1]
```diff
@@ -34,47 +38,27 @@ $( document ).ajaxComplete(function() {
...
-  $(this).siblings('i.show-journal').prop('hidden', false);
-});
 
-$( '#search-panel' ).on( 'click', 'i.show-year', function( event ) {
-  $(this).siblings('div.year')
-      .children('div.toggleable-offset').removeClass('offset-md-4');
-  $(this).siblings('div.year').children('div.year').prop('hidden', false);
-  $(this).siblings('div.year').children('div.year')
-      .children('input').prop('disabled', false);
-  $(this).siblings('div.year').children('div.year')
```
Comment (CBrobsto) [inline public/javascripts/database.js:62]: Can someone remind me why this chunk of code was removed?

**Review by CBrobsto — APPROVED** [review]
See comments in the conversation tab. I ran a basic high-level test and did not find any major problems. Jest produced failed tests but they fail the same tests in Master as well.

File: public/javascripts/database.js (hunk 1 above)
Comment (ccqian-asu) [reply public/javascripts/database.js:62]: @JeffFallis Removed it in a commit: "Removed jquery regarding 'year' group that was removed in an earlier commit"

I think we refactored how/where that element shows up

**Review by ccqian-asu — APPROVED** [review]

**Pull request merged**